hschain-utxo-lang/src/Hschain/Utxo/Lang/Sigma/EllipticCurve.hs

```diff
@@ -46,6 +47,8 @@ class ( ByteRepr (ECPoint   a)
   generateScalar    :: IO (ECScalar a)
   fromGenerator     :: ECScalar  a -> ECPoint  a
   fromChallenge     :: Challenge a -> ECScalar a
+  isIdentity        :: ECPoint a -> Bool
+  groupGenerator    :: ECPoint a
```

Comment:
What is groupGenerator? What conditions should point satisfy? How is it related to `fromGenerator`?